packages/evo/src/wrap/WrapClient.ts

```diff
@@ -179,7 +179,53 @@ export class WrapClient extends PolywrapClient {
         "extname": (args: any) => path.extname(args.path),
         "format": (args: any) => path.format(args.pathObject),
         "parse": (args: any) => path.parse(args.path)
-      })));
+      })))
+      .setPackage("plugin/websearch", PluginPackage.from(module => ({
+        "search": async (args: { query: string }) => {
+          const axiosClient =  axios.create({ baseURL: 'https://api.search.brave.com/res/v1/web' });
+
+          if (!process.env.BRAVE_API_KEY) {
```

Comment:
NIT: we're currently initializing environment variables in a uniform way like so https://github.com/polywrap/evo.ninja/blob/a70fd70868f15a0e381b69d24c86ac142cadd486/packages/agent-utils/src/sys/env.ts#L1

Since the base `Env` class is within `agent-utils` and this is within the `evo` package, I'd recommend the following `class Env extends BaseEnv` and then exporting this from the `evo` package instead of the `agent-utils` version.

Reply:
Yep. Changed the code to use that + added an input in the Dojo for the same purpose.